### PR TITLE
Replace returnVals by returnChanges

### PR DESCRIPTION
**Pull request by danielmewes**
This should fix https://github.com/rethinkdb/rethinkdb-example-nodejs/issues/11 and https://github.com/rethinkdb/rethinkdb-example-nodejs/issues/12 .

@deontologician could you review please?

### CODE REVIEW

File: todo-angular-koa/app.js [hunk 1]
```diff
@@ -84,7 +84,7 @@ function* update(next) {
             throw new Error("The todo must have a field `id`.");
         }
 
-        var result = yield r.table('todos').get(todo.id).update(todo, {returnVals: true}).run(this._rdbConn);
+        var result = yield r.table('todos').get(todo.id).update(todo, {returnChanges: true}).run(this._rdbConn);
         this.body = JSON.stringify(result.new_val);
```
Comment (anthonyettinger) [inline todo-angular-koa/app.js]: This line needs updating I believe.

Comment (danielmewes) [inline todo-angular-koa/app.js]: Which part of it?

Comment (anthonyettinger) [inline todo-angular-koa/app.js]: Sorry. I believe it should be `result.changes[0].new_val` -- but i'm not sure of the impact of only looking at `[0]`

Comment (danielmewes) [inline todo-angular-koa/app.js]: You are right! Didn't think about it enough. I opened https://github.com/rethinkdb/rethinkdb-example-nodejs/issues/14 so we don't forget changing that.

**Pull request merged**